generate-stackbrew-library.sh

```diff
@@ -62,6 +31,48 @@ dirCommit() {
 	)
 }
 
+getArches() {
+	local repo="$1"; shift
+	local officialImagesUrl='https://github.com/docker-library/official-images/raw/master/library/'
+
+	eval "declare -A -g parentRepoToArches=( $(
+		find -name 'Dockerfile' -exec awk '
+				toupper($1) == "FROM" && $2 !~ /^('"$repo"'|scratch|.*\/.*)(:|$)/ {
+					print "'"$officialImagesUrl"'" $2
+				}
+			' '{}' + \
+			| sort -u \
+			| xargs bashbrew cat --format '[{{ .RepoName }}:{{ .TagName }}]="{{ join " " .TagEntry.Architectures }}"'
+	) )"
+}
+getArches 'docker'
+
+versionArches() {
+	local version="$1"; shift
+	local variant="${1:-}"
+	local selector='dockerUrl'
+	if [[ "$variant" = *rootless ]]; then
+		selector='rootlessExtrasUrl'
+	fi
+
+	local parent parentArches
+	parent="$(awk 'toupper($1) == "FROM" { print $2 }' "$version/Dockerfile")"
+	parentArches="${parentRepoToArches[$parent]:-}"
+
+	comm -12 \
+		<(
+			version="$version" jq -r '
+				.[env.version].arches | to_entries[]
+				| select(.value.'"$selector"')
+				| .key
+			' versions.json | sort
+		) \
+		<(xargs -n1 <<<"$parentArches" | sort)
+}
+
+# sort version numbers with highest first
+#IFS=$'\n'; versions=( $(echo "${versions[*]}" | sort -rV) ); unset IFS
```

Comment:
We should probably delete this comment and just copy what Go has: https://github.com/docker-library/golang/blob/ee2d52a7ad3e077af02313cd4cd87fd39837412c/generate-stackbrew-library.sh#L26-L27